docs/python_functions.md

```diff
@@ -70,10 +89,10 @@ def shift_left(x):
     return 9e8c617fe2e0d524469d75f43edb1ff91f9a5387af6c444017ddcd194c983aed(x, 2) 
```

Comment:
Is the hash being detected as a secret of sorts?

Reply:
We can change the hash, as long as something is there

Reply:
Changed the code block to `text` and added a note on the syntax validity (this is intentional)